core/src/main/kotlin/no/ntnu/beardblaster/ui/Elements.kt

```diff
@@ -40,6 +40,13 @@ fun bodyLabel(text: String, labelStyle: String = LabelStyle.Body.name): Label {
     }
 }
 
+fun smallBodyLabel(text: String, labelStyle: String = LabelStyle.Body.name): Label {
```

Comment:
Endre heller den originale "body" med variabel font-scale?

```suggestion
fun bodyLabel(text: String, labelStyle: String = LabelStyle.Body.name, fontScale: Float  = 1.5f): Label {
 return scene2d.label(text, labelStyle) {
        setAlignment(Align.center)
        setFontScale(fontScale)
    }
}
```